src/main/java/io/debezium/connector/yugabytedb/consistent/Merger.java

```diff
@@ -38,7 +38,7 @@ public Merger(List<String> tabletList) {
     public synchronized void addMessage(Message message) {
         assert message.record.getRowMessage().getOp() != CdcService.RowMessage.Op.DDL;
 
-        setTabletSafeTime(message.tablet, message.commitTime);
+        setTabletSafeTime(message.tablet, message.commitTime, message);
```

Comment:
Change to print the message as well, in case tablet safetime verification fails.